api4/post.go

```diff
@@ -67,7 +67,15 @@ func createPost(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	c.App.SetStatusOnline(c.App.Session.UserId, false)
+	setOnline := r.URL.Query().Get("set_online")
+	setOnlineBool, err2 := strconv.ParseBool(setOnline)
```

Comment:
I would differ between an empty string and an non empty string here. Then we could log an error if an non empty, but malformed string is received.